src/S2_stacking.py

```diff
@@ -37,339 +35,345 @@
 #########PARAMETER SECTION##############
 ########################################
 
-# absolute path parameters
-rootpath  = os.path.join(os.path.expanduser('~'), 'Documents/SCAL') # root path for this data processing
-CCFDIR    = os.path.join(rootpath,'CCF')                            # dir where CC data is stored
-STACKDIR  = os.path.join(rootpath,'STACK')                          # dir where stacked data is going to
-locations = os.path.join(rootpath,'RAW_DATA/station.txt')           # station info including network,station,channel,latitude,longitude,elevation
-if not os.path.isfile(locations): 
-    raise ValueError('Abort! station info is needed for this script')
 
 # define new stacking para
 keep_substack= False                                                # keep all sub-stacks in final ASDF file
 flag         = True                                                 # output intermediate args for debugging
-stack_method = 'linear'                                             # linear, pws, robust, nroot, selective, auto_covariance or all
 
 # new rotation para
 rotation     = True                                                 # rotation from E-N-Z to R-T-Z 
 correction   = False                                                # angle correction due to mis-orientation
-if rotation and correction:
-    corrfile = os.path.join(rootpath,'meso_angles.txt')             # csv file containing angle info to be corrected
```

Comment:
I see a hardcoded file here that was quite specific to a project we had. Could we write this filename as a separate variable input. it matters a lot for borehole and offshore stations where there is less control on how the sensor is oriented. For a later modification: we should also have this file input as a CSV and use dataframes instead for metadata.

Reply:
Got it. I've created an issue for this task, so that this PR maintaince its single focus.